backend.py

```diff
@@ -288,7 +273,6 @@ def get_active_robots(self):
 def get_start_tiles(board):
     """
     Get start tiles for robots.
```

Comment:
Ty mezery v docstringach jsou schvalne: https://github.com/PyLadiesCZ/roboprojekt/issues/28

Reply:
? Teď nechápu k čemu se to vztahuje co je kde blbě?

Reply:
Jakože to mám zmáčknout k sobě na míň řádků? :)

Reply:
Asi budu potřebovat přímo říct kde mezery jo a kde ne :)

Reply:
Nu, tenhle Pull Request maže prázdné řádky.
Když už tam mezery jsou, tak je nevyndávej bez důvodu :)

Reply:
Doufám že tam jsou vrácené dobře protože to mám u sebe už celé zase předělané, smazané větve. Vůbec jsem je úmyslně nemazala, nevím o tom .